config/jenkins_infra.ci.jenkins.io.yaml

```
@@ -288,11 +288,15 @@ controller:
                     EOF
                     systemctl daemon-reload
                     systemctl restart docker
+                    export DEFAULT_JDK=/opt/jdk-11
+                    update-alternatives --install /usr/bin/java java "${DEFAULT_JDK}/bin/java" 2000
+                    echo "JAVA_HOME=${DEFAULT_JDK}" >> /etc/environment
```

```suggestion
                    echo "JAVA_HOME=^${DEFAULT_JDK}" >> /etc/environment
```

Need to escape `$` sign on JCasc if you want it to be litteral